PowerEditor/src/winmain.cpp

```diff
@@ -101,6 +105,21 @@ void stripIgnoredParams(ParamVector & params)
 			auto nextIt = std::next(it);
 			if ( nextIt != params.end() )
 			{
+				PWSTR ignoredString = wcsstr(pCmdLine, nextIt->c_str());
+				if ( ignoredString != nullptr )
+				{
+					ignoredString += nextIt->length();
+
+					// Advance to the first non-whitespace and not quotation mark character
+					while ( iswspace( *ignoredString ) || *ignoredString == L'"' )
+					{
+						if ( *ignoredString == '\0' )

```

Comment:
Does this do anything? I don't see either while condition letting a NULL character through.

Reply:
Good point - according to https://en.cppreference.com/w/cpp/string/wide/iswspace null terminator is not considered whitespace, and thus it should be out of this loop by then. Will correct tomorrow.